public/index.html

```diff
@@ -7,37 +7,14 @@
     <meta name="theme-color" content="#000000" />
     <meta
       name="description"
-      content="Web site created using create-react-app"
+      content="Manage your productivity"
```

Comment:
[TODO] We should add additional metadata to the page, as necessary.

Some related material: https://ahrefs.com/blog/seo-meta-tags/